androidApp/src/main/java/dev/johnoreilly/starwars/androidApp/MainActivity.kt

```diff
@@ -76,7 +92,16 @@ fun MainLayout() {
     }
 }
 
-
+@Composable
+private fun rememberStarWarsRepository(): StarWarsRepository {
+    val context = LocalContext.current.applicationContext
+    val sqlNormalizedCacheFactory = remember { SqlNormalizedCacheFactory(context, "swapi.db") }
+    val repo = remember { StarWarsRepository(dbCacheFactory = sqlNormalizedCacheFactory) }
```

Comment:
Nitpicking: could you save one "remember" here and put everything (cache + repo) in the same "remember" ? 

```kotlin
val repo = remember { 
   StarWarsRepository(dbCacheFactory = SqlNormalizedCacheFactory(context, "swapi.db")) 
}
```

Reply:
Yep - but no longer relevant after Koin refactor.